docs/modules/ROOT/pages/about-browser.adoc

```diff
@@ -1,37 +1,26 @@
 [[about-browser]]
 = About Neo4j Browser
-
-[abstract]
---
-This section describes the purpose of Neo4j Browser and what it is used for.
---
+:description: This section describes the purpose of Neo4j Browser and its high-level capabilities.
 
 [[browser-purpose]]
 == Purpose of Neo4j Browser
 
-Neo4j Browser is a rich interface for querying and visualising a Neo4j Database.
-It provides an interactive experience for writing, editing and evaluating the results of link:https://neo4j.com/docs/cypher-manual/current/introduction/[Cypher] queries to explore your graph data.
+Neo4j Browser is a rich interface for querying and visualising data in Neo4j.
```

Comment:
Neo4j Browser is a rich interface for querying Neo4j and visualizing data results.